Shawntru/Rotten-Tomatillos
# Fix error handling and error page

## Is this a fix or a feature?
Fix

## What is the change?
Changed error page and error handling for 500 error, missing home page movies data and missing single movie data

## What does it fix?
Error handling

## Where should the reviewer start?
App.js line 21
ErrorPage.js line 3
ErrorPage.scss line 12
MoviePreview.js line 33
MoviePreview.scss line 85
apiCalls.js line 1

## How should this be tested?
Running npm start will generate a full functional website, if movies or single movie info is missing the User will see an error page

## Relevant Ticket(s)
